app/containers/CompanyReview/index.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import * as Yup from 'yup';
+import { createStructuredSelector } from 'reselect';
+
+import injectReducer from 'utils/injectReducer';
+import injectSaga from 'utils/injectSaga';
+import CompanyReviewComponent from 'components/CompanyReview';
+import { selectLoggedUserId } from 'containers/App/selectors';
+import { selectErrorMessage, selectHasReviewed } from './selectors';
+import { addReview } from './actions';
+import reducer from './reducer';
+import saga from './saga';
+
+const config = {
+  initialValues: {
+    comment: '',
+  },
+  validationSchema: Yup.object().shape({
+    comment: Yup.string('').required('This field is required.'),
+  }),
+  fields: [
+    {
+      type: 'input',
+      options: {
+        type: 'comment',
+        label: 'Comment',
+        name: 'comment',
+      },
+    },
+    {
+      type: 'button',
+      options: {
+        text: 'SUBMIT',
+        name: 'submitButton',
+      },
+    },
+  ],
+};
+
+class CompanyReview extends React.Component {
+  state = {
+    canComment: false,
+    rating: 0,
+  };
+
+  onRating(rating) {
+    const { state } = this;
+    this.setState({
+      canComment: true,
+      rating,
+      ...state,
+    });
+  }
+
+  render() {
+    return (
+      <CompanyReviewComponent
+        onSubmit={values =>
+          this.props.onSubmit({
```

Comment:
When you use props values, define them as the first thing in the function, like this: `const { onSubmit, loggedUserId, companyId, rating, ... } = this.props;`.